src/integrationTest/java/uk/gov/hmcts/reform/pdf/service/integration/GeneratedPDFContentV2Test.java

```diff
@@ -44,30 +65,54 @@ private static String textContentOf(byte[] pdfData) throws IOException {
         }
     }
 
-    @SuppressWarnings("unchecked")
+    @After
+    public void verifyAuth() {
+        verify(authorisationService).authorise("some-service-auth-header");
+    }
+
     @Test
     public void shouldCreateExpectedPdfFromPlainHtmlTemplate() throws Exception {
-        assertThatGeneratedPdfContains("<html><body>Hello!</body></html>", Collections.EMPTY_MAP, "Hello!");
+        HttpServletResponse response = getResponse("<html><body>Hello!</body></html>", Collections.emptyMap());
+        assertGeneratedPdfContains(response, "Hello!");
     }
 
     @Test
     public void shouldCreateExpectedPdfFromPlainTwigTemplateAndPlaceholders() throws Exception {
-        Map<String, Object> values = ImmutableMap.of("hello", "World!");
-        assertThatGeneratedPdfContains("<html>{{ hello }}</html>", values, "World!");
+        HttpServletResponse response = getResponse("<html>{{ hello }}</html>", ImmutableMap.of("hello", "World!"));
+        assertGeneratedPdfContains(response, "World!");
     }
 
-    private void assertThatGeneratedPdfContains(String template, Map<String, Object> values, String expectedText)
-        throws Exception {
+    @Test
+    public void shouldFailAuthorisationWhenServiceThrowsException() throws Exception {
+        Response feignResponse = Response.builder()
+            .headers(Collections.emptyMap())
+            .status(HttpStatus.UNAUTHORIZED.value())
+            .build();
+        FeignException exception = FeignException.errorStatus("oh no", feignResponse);
+
+        when(authorisationService.authorise("some-service-auth-header")).thenThrow(exception);
```

Comment:
perhaps `anyString()` instead of `"some-service-auth-header"`.
it's not clear that it's the token that's being sent and we don't really care what's being sent.